### PR TITLE
Logs in modal window for pods and lambdas fixed

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- append `splitView` node param to enforce compact view for logs in modals (pods & lambda list)
- add `Show logs` menu entry  basing on `loki` BM existance
- fix log labels for pods 

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/4711

### CODE REVIEW

File: core/src/app/content/namespaces/operation/pods/pods.component.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class PodsComponent extends AbstractGraphqlElementListComponent
     handler.showLogs = (entry: any) => {
       luigiClient
         .linkManager()
-        .withParams({ pod: entry.name, namespace: this.currentNamespaceId })
+        .withParams({ app: entry.labels.app, namespace: this.currentNamespaceId, splitViewMode: 'true' })
```
Comment (sjanota) [inline core/src/app/content/namespaces/operation/pods/pods.component.ts]: `app` label AFAIK is optional. Shouldn't we rely on `pod`?

Comment (kwiatekus) [reply core/src/app/content/namespaces/operation/pods/pods.component.ts]: the problem is ... the pod label is no longer there

Comment (kwiatekus) [reply core/src/app/content/namespaces/operation/pods/pods.component.ts]: ... but still, I have changes the logic here.. 
using app label if present and additionally pod-template-hash label, if present

File: core/src/app/content/namespaces/operation/pods/pods.component.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { PodsHeaderRendererComponent } from './pods-header-renderer/pods-header-
 
 import * as luigiClient from '@kyma-project/luigi-client';
 import { GraphQLClientService } from 'shared/services/graphql-client-service';
+import { identifierModuleUrl } from '@angular/compiler';
```
Comment (y-kkamil) [inline core/src/app/content/namespaces/operation/pods/pods.component.ts]: what's it for here?

**Pull request merged**